README.md

```diff
@@ -1,3 +1,34 @@
-# Extension Activity Monitor
+# Extension Activity Monitor [![Build Status](https://travis-ci.org/mozilla/extension-activity-monitor.svg?branch=master)](https://travis-ci.org/mozilla/extension-activity-monitor) [![Coverage Status](https://coveralls.io/repos/github/mozilla/extension-activity-monitor/badge.svg?branch=master)](https://coveralls.io/github/mozilla/extension-activity-monitor?branch=master)
 
-TBD
+This is a privileged extension for Firefox that monitors the activities of other installed extensions. The extension uses a privileged API called `activityLog` API to retrieve logs from other installed extensions.
+
+## Installation
+
+### Prerequisite
+
+- Clone / download the respository in your local machine.
+- Install [Firefox Nightly](https://www.mozilla.org/en-US/firefox/all/#product-desktop-nightly) to run this privileged extension.
```

Comment:
Nit:
- Clone ...
- Install ...
- [optional] `npm ci`, to install dev dependency to develop a patch or to run the extension using the web-ext dependency

Reply:
Added it :-)